### PR TITLE
Add date, datetime casts

### PR DESCRIPTION
**Pull request by joeirimpan**
@prakashpp Please review

### CODE REVIEW

File: fulfil_client/model.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 and better
 """
 import functools
+from datetime import datetime
```
Comment (prakashpp) [inline fulfil_client/model.py]: from datetime import date

File: fulfil_client/model.py [hunk 2]
```diff
@@ -82,6 +83,13 @@ def __init__(self, *args, **kwargs):
         super(FloatType, self).__init__(*args, **kwargs)
 
 
+class DateType(BaseType):
```
Comment (prakashpp) [inline fulfil_client/model.py]: This can be DateTime
Date

**Review by prakashpp — CHANGES_REQUESTED** [review]

File: fulfil_client/model.py (hunk 1 above)
Comment (joeirimpan) [reply fulfil_client/model.py]: fixed

File: fulfil_client/model.py (hunk 2 above)
Comment (joeirimpan) [reply fulfil_client/model.py]: fixed

**Review by prakashpp — APPROVED** [review]

**Pull request merged**